lib/DDG/Spice/IndegoBikeshareLocations.pm

```diff
@@ -0,0 +1,34 @@
+package DDG::Spice::IndegoBikeshareLocations;
```

Comment:
I wouldn't bet on just one system per city (even though, the examples I can think of match that logic). Is package name length really an issue?

Now that you mention this, I actually think I'd like something similar to: `BikeSharing::NYCCitiBike` (City + System Name)

Apart from the namespace, it would also be nice to move inside the same root level dir